domains/vaibhav.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Vaibhav-Kashyap",
+    "email": "",
+    "discord": "Vaibhav Kashyap#5747"
+  },
+  "record": {
+  
```

Comment:
Please add at least 1 record.

Reply:
How do I edit it? or do I have to make a new pull request?

Reply:
Nvm There I fixed it.